doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_reg.rst

```diff
@@ -8,6 +8,29 @@ Regulator implementations should declare a :c:struct:`bt_mesh_light_ctrl_reg` st
 
 For an example of a regulator implementation, see :ref:`bt_mesh_light_ctrl_reg_spec_readme`.
 
+Using the regulator
+*******************
+
+As this module implements an abstract regulator, it cannot be used on its own.
+A regulator implementation will provide a :c:struct:`bt_mesh_light_ctrl_reg` instance, and this is used as the interface to the regulator.
+Users of a regulator will call ``init``, ``start``, and ``stop`` functions from this instance, as well as :c:func:`bt_mesh_light_ctrl_reg_input_set`.
+
+To get output from the regulator, set the ``update`` callback on the struct.
+There is a field in the struct for ``user_data`` which can be supplied by the user and will be available through the regulator pointer passed to the ``update`` callback.
+
+Implementing a regulator
+************************
+
+To implement a regulator using this generic interface, declare a :c:struct:`bt_mesh_light_ctrl_reg` struct, and set the ``init``, ``start``, and ``stop`` fields to implementations of these functions.
+To get the current target value for the regulator, supplied by the user of the regulator, use :c:func:`bt_mesh_light_ctrl_reg_input_get`, which returns the requested target value.
+The value returned is interpolated linearly over the transition time, if a transition time is requested by the regulator user.
+
+The ``init`` should perform any necesary steps to initialize the implementation, such as initializing interrupt handlers or timers, but not start the regulator.
+
+The ``start`` and ``stop`` functions should start and stop the regulator, after it has been initialized by a call to ``init``.
+
+When the regulator is running, it should repeatedly call the ``update`` function on the :c:struct:`bt_mesh_light_ctrl_reg` struct, which is supplied by the user.
```

Comment:
```suggestion
When the regulator is running, it will repeatedly call the ``update`` function on the :c:struct:`bt_mesh_light_ctrl_reg` struct, which is supplied by the user.
```